content/guides/analog-discovery/_index.md

```diff
@@ -0,0 +1,886 @@
+2-      GND
+0       in0
+1       in1
+2       in2
+3       in3
+9       in5
+10      in6
+GND     GND
+```
+
+If you also want to run the oscillator frequency vs core voltage characterization, you'll need to
+
+   * depopulate F2 from the demoboard
+   * wire *V+* from the AD3 to the 1V8 pin on the demoboard top header
+
+
+This is optional as you may skip the test at the prompt
+{{< figure src="images/47.png" title="Test run prompt" >}}
+
+or simply get the boring result of no-frequency-variation (since the core voltage won't actually be set by the AD3).
+
+## How to run
+
+
+Assuming you've wired everything up, these are the steps to run the test suite.  Following the entire sequence is only important if you've set things to drive the core supply from the AD3, but it doesn't hurt even if you haven't
+
+   1. connect the AD3 to USB
+   2. open the WaveForms program and load the workspace file
+   3. switch to the *Supplies* tab, ensure Positive supply is set to 1.8V and press the "play" button on that tab to enable the core voltage.  Make sure both the "master enable" and "positive supply" have green checkmarks;
+   4. switch to the *Script* tab, find the *powerupHighZ* tab in there and press its "play" button to run it, then click *Yes* to the "Do you want to put all DIO into High-Z" prompt
+   5. connect the demoboard to USB
+   6. press "play" button on the *demoboardsetup* script tab, and specify the correct "com port" (ttyACM0 is the default, and usually valid, under linux)
+   7. press "play" on the *powerupHighZ*, again... but this time click "No" to the first prompt and then "Yes" to "Do you wish to restore Patterns signals to PP?"
+
+Now, everything should be in a good state, and you shouldn't have to go through that dance again unless you power down.
+
+Clicking the "play" button on the *tests* tab in Script will run the tests--simply follow the prompts.
+
+{{< figure src="images/49.png" title="DAC plot prompt" >}}
+
+
+
+
+
+### The Oscilloscope
+
+When you launch waveforms you get a welcome screen with all the available instruments.
+
+{{< figure src="images/2.png" title="Welcome Screen" >}}
+
+If you click on one of them, for example the oscilloscope, it will open up a new tab with that instrument panel.
+
+{{< figure src="images/3.png" title="Oscilloscope" >}}
+
+The oscilloscope has two channels, aptly named 1 and 2.  Each channel has a + and a - pin.  In this case, the 
+
+ * 1+ is connected to the *m33* pin in the top header (the ring oscillator output);
+ * 2+ is connected to the *m35* pin (the DAC output); and 
+ * both 1- and 2- are connected to GND pads.
+ 
+
+
+{{< figure src="images/ad3scopeconn.jpg" title="AD3 Scope connections" >}}
+
+## Ring Oscillator 
+
+For this to work, the analog test project must loaded and configured.  This could be done through the [commander app](https://commander.tinytapeout.com/), but it's sometimes quicker to connect to the REPL directly via a serial terminal and use the SDK to:
```

Comment:
worth mentioning there are 2approaches taken here, the script talks directly to the board and you don't use the commander. the walkthrough uses more manual steps with the commander to configure things